website/docs/r/ssm_maintenance_window_task.html.markdown

```diff
@@ -140,7 +140,7 @@ The following arguments are supported:
 * `window_id` - (Required) The Id of the maintenance window to register the task with.
 * `max_concurrency` - (Required) The maximum number of targets this task can be run for in parallel.
 * `max_errors` - (Required) The maximum number of errors allowed before this task stops being scheduled.
-* `task_type` - (Required) The type of task being registered. The only allowed value is `RUN_COMMAND`.
+* `task_type` - (Required) The type of task being registered. Valid options are `AUTOMATION`, `LAMBDA`, `RUN_COMMAND` or `STEP_FUNCTIONS`.
```

Comment:
Small nitpick otherwise looks great 👍 :
```suggestion
* `task_type` - (Required) The type of task being registered. Valid values: `AUTOMATION`, `LAMBDA`, `RUN_COMMAND` or `STEP_FUNCTIONS`.
```